<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="0" failed="6" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-26T17:07:11 IST" name="Surefire suite" finished-at="2023-05-26T17:10:03 IST" duration-ms="172394">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.runnerTest@4659191b]" name="runScenario" class="io.cucumber.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2023-05-26T17:07:11 IST" name="Surefire test" finished-at="2023-05-26T17:10:03 IST" duration-ms="172394">
      <class name="testRunner.runnerTest">
        <test-method is-config="true" signature="setUpClass(org.testng.ITestContext)[pri:0, instance:testRunner.runnerTest@4659191b]" started-at="2023-05-26T17:07:11 IST" name="setUpClass" finished-at="2023-05-26T17:07:12 IST" duration-ms="1432" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@65db548]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.runnerTest@4659191b]" started-at="2023-05-26T17:07:13 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-05-26T17:07:52 IST" duration-ms="39143" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Validate the final amount paid for Hotel"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Validate Filter and Final Amount"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at utilities.Hooks.afterScenario(Hooks.java:38)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.runnerTest@4659191b]" started-at="2023-05-26T17:07:52 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-05-26T17:08:26 IST" duration-ms="34301" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Validate the filter applied on Hotel"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Validate Filter and Final Amount"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at utilities.Hooks.afterScenario(Hooks.java:38)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.runnerTest@4659191b]" started-at="2023-05-26T17:08:26 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-05-26T17:08:45 IST" duration-ms="18789" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Register to MMT to list your own hotel"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Register to MMT"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at utilities.Hooks.afterScenario(Hooks.java:38)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.runnerTest@4659191b]" started-at="2023-05-26T17:08:45 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-05-26T17:09:08 IST" duration-ms="23337" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Login to MMT with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login to MMT"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at utilities.Hooks.afterScenario(Hooks.java:38)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.runnerTest@4659191b]" started-at="2023-05-26T17:09:08 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-05-26T17:09:33 IST" duration-ms="24512" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Login to MMT with valid credentials"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Login to MMT"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at utilities.Hooks.afterScenario(Hooks.java:38)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method signature="runScenario(io.cucumber.testng.PickleWrapper,io.cucumber.testng.FeatureWrapper)[pri:0, instance:testRunner.runnerTest@4659191b]" started-at="2023-05-26T17:09:33 IST" name="runScenario" description="Runs Cucumber Scenarios" data-provider="scenarios" finished-at="2023-05-26T17:10:03 IST" duration-ms="30034" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA["Open and check MMT twitter and facebook"]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA["Verify MMT Social Media Sites"]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.fail(Assert.java:115)
at utilities.Hooks.afterScenario(Hooks.java:38)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- runScenario -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:testRunner.runnerTest@4659191b]" started-at="2023-05-26T17:10:03 IST" name="tearDownClass" finished-at="2023-05-26T17:10:03 IST" duration-ms="771" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- testRunner.runnerTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
